AprilSylph/XKit-Rewritten
# Implement Scroll to Bottom on Timeline V2

### Description


This implements Scroll to Bottom on V2 timelines.

As noted in the linked issue, on V2 timelines, the loader element we need to detect so that we keep scrolling conditionally:

- Isn't the last child of its parent anymore (and doesn't have a classname iirc?)
- Doesn't exist unless the timeline is actively loading content

Thus, this:

- Fixes the selector
- Implements new logic: scrolling will stop after a short delay unless a loader is added during the delay window. (This is basically equivalent to "debounce the stopscrolling function" in some meaningful sense, though I believe literally doing that has some bad edge cases. Didn't think this through much; if that works we should do it so the code is easier to understand.)

Resolves #1491.

Merge conflicts spectacularly with #977, but what can you do, I guess. (#1492, is what.)

### Testing steps


not formally tested as of this pr description update; should be tested on following/followers/activity as well as post timelines

edit: tested (works, stops at end) on:
 - following
 - followers
 - activity
